Extension/src/LanguageServer/configurations.ts

```diff
@@ -737,8 +737,16 @@ export class CppProperties {
 
     private resolveAndSplit(paths: string[] | undefined, defaultValue: string[] | undefined, env: Environment, glob = false): string[] {
         const result: string[] = [];
+        let recursiveDoubleStar: boolean = false;
+        let recursiveForwardSlash: boolean = false;
+        let recursiveForwardSlashSingleStar: boolean = false;
+        let recursiveDoubleBackwardSlashDoubleStar: boolean = false;
         if (paths) {
             paths = this.resolveDefaults(paths, defaultValue);
+            recursiveDoubleStar = paths[0].endsWith('**') && !paths[0].endsWith('\\**');
```

Comment:
This needs to be moved below into the `for (let res of result)` loop. Test case is multiple paths in includePath.

Reply:
The issue with that is that the strings get normalized so I can't check correctly for correct path endings

Reply:
Never mind, I have a fix.